.golangci.yml

```diff
@@ -87,6 +87,12 @@ issues:
     # checks from err113 are useful.
     - "err113: do not define dynamic errors.*"
   exclude-rules:
+    # Clone uses unchecked type conversions after invocations of proto.Clone
+    # which is safe because of the guarantees of proto.Clone and the static
+    # type of the input argument.
+    - path: parser/clone(_test)?.go
+      linters:
+        - errcheck
```

Comment:
We are using `//nolint:errcheck` in some places and this config disables it for the files entirely. Do we want to just use the nolint comments so we can be specific about which places should disable linters?

Reply:
Yeah, I can change these. I was thinking maybe there were a lot in these files, so it was simpler this way. But that's not really true. I'll change it.